src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-console.log('hello mate academy!');
+const bigImg = document.querySelector('.gallery__large-img');
+
+function replaceImg(e) {
+  const { className: elemClass } = e.target;
+
+  if (elemClass.includes('gallery__thumb')
+    || elemClass.includes('list-item__link')) {
+    e.preventDefault();
+
+    const { href: elemHref } = e.target.closest('[href]');
+
+    bigImg.setAttribute('src', elemHref);
+  }
+}
+
+document.body.addEventListener('click', replaceImg);
```

Comment:
```suggestion
```suggestion
document.body.addEventListener('click', replaceImg);
```
document.body.addEventListener('click', onImageClick);
```